MoviesApp/MovieDetail/Controller/MovieDetailViewController.swift

```diff
@@ -49,7 +49,7 @@ final class MovieDetailViewController: UIViewController {
     }
 
     private func failure() {
-        showMessageAlert(title: "Нет подключения к интернету", message: "Попробовать еще?") {
+        showLoadingErrorAlert(title: "Нет подключения к интернету", message: "Попробовать еще?") {
```

Comment:
```suggestion
        showLoadingErrorAlert(title: "Нет подключения к интернету", message: "Попробовать еще?") {
```
 "Нет подключения к интернету", message: "Попробовать еще?" в константу

Reply:
```suggestion
        showLoadingErrorAlert(title: "Нет подключения к интернету", message: "Попробовать еще?") { [weak self] in
```